docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "MemoryArrayCreationOverflow",
+        "summary": "The array length when creating a dynamic memory array is unrestricted and can overflow, potentially resulting in overlapping memory regions.",
```

Comment:
"The creation of very large memory arrays can result in overlapping memory regions and thus memory corruption"